src/javascripts/ng-admin/Crud/column/maColumn.js

```diff
@@ -3,7 +3,22 @@
 define(function (require) {
     'use strict';
 
-    function maColumn($location, $anchorScroll, $compile, Configuration) {
+    var _ = require('lodash');
```

Comment:
I'm not sure we should rely on that. I actually use only `map()` (ES5), which we could assume is always present. For browsers without ES5 support I believe it's the developer's responsibility to use a shim. What do you think?